core/src/com/unciv/logic/map/UnitMovementAlgorithms.kt

```diff
@@ -336,9 +336,9 @@ class UnitMovementAlgorithms(val unit: MapUnit) {
         return getDistanceToTiles().containsKey(destination)
     }
 
-    fun getReachableTilesInCurrentTurn(): Sequence<TileInfo> {
+    fun getReachableTilesInCurrentTurn(attacking: Boolean=false): Sequence<TileInfo> {
```

Comment:
HMMMM
There must be a better solution than adding another input parameter
Is this for attackable tiles? If so we can put the logic there

Reply:
This is specifically so that we can paint the attackable tiles since that code utilizes this to give a subset of tiles to check

Reply:
Then that code should, if the unit cannot move, use neighbors instead
We always prefer to move die to more specific areas rate than more general ones

Reply:
Done